packages/saber-plugin-images/README.md

```diff
@@ -0,0 +1,63 @@
+# saber-plugin-images
+
+Allows the generation of responsive images, lazy-loading and all that good stuff.
+
+## Install
+
+```bash
+yarn add saber-plugin-images
+```
+
+## Usage
+
+In your `saber-config.yml`:
+
+```yml
+plugins:
+  - resolve: saber-plugin-images
+    options:
+      sizes:
+        - 1000
+        - 800
+        - 500
+      placeholder: true
+```
+
+All loader options can be found [here](https://github.com/herrstucki/responsive-loader#options).
+
+As Saber currently does not allow plugins to add Vue components, you'll need to add the following to your `saber-browser.js`:
+
+```js
+import SaberImage from 'saber-plugin-images/SaberImage'
+
+export default ({ Vue }) => {
+  Vue.use(SaberImage, {
+    error: require('./images/error.png')
+  })
+}
+```
```

Comment:
why not add a `saber-browser.js` in this plugin?

Reply:
Oh, I didn't know that was possible :smile: